iquip/explorer.py

```diff
@@ -0,0 +1,33 @@
+"""
+App module for showing the experiment list and opening an experiment.
+"""
+
+from typing import Optional, Tuple
+
+from PyQt5.QtCore import QObject
+from PyQt5.QtWidgets import QLabel, QVBoxLayout, QWidget
+
+from qiwis import BaseApp
+
+class ExplorerFrame(QWidget):
+    """Frame for showing the experiment list and opening an experiment."""
+    def __init__(self, parent: Optional[QObject] = None):
```

Comment:
`QWidget`'s [constructor](https://doc.qt.io/qt-5/qwidget.html#QWidget) takes `QWidget parent`, not `QObject`!

Reply:
Wow, you're right! Then, why did we set the parents of the example apps as `QObject` in `qiwis`..?
I think we were missing it.